pkg/collector/corechecks/ebpf/probe/ebpfcheck/probe.go

```diff
@@ -45,12 +45,13 @@ const maxMapsTracked = 20
 
 // Probe is the eBPF side of the eBPF check
 type Probe struct {
-	statsFD       io.Closer
-	coll          *ebpf.Collection
-	perfBufferMap *ebpf.Map
-	ringBufferMap *ebpf.Map
-	pidMap        *ebpf.Map
-	links         []link.Link
+	statsFD           io.Closer
+	coll              *ebpf.Collection
+	perfBufferMap     *ebpf.Map
+	ringBufferMap     *ebpf.Map
+	pidMap            *ebpf.Map
+	links             []link.Link
+	entryCountBuffers entryCountBuffers
```

Comment:
variable named the same as the type is confusing

Reply:
changed!